dockerGrader/testServer.py

```diff
@@ -128,7 +115,6 @@ def sendToDocker(project, netId):
 
     # run tests inside a docker container
     image = 'python:3.7-stretch' # TODO: find/build some anaconda image
-
     cmd = ['docker', 'run',                           # start a container
```

Comment:
Maybe using --rm would be an elegant way to exit fast?

https://www.powercms.in/blog/how-automatically-delete-docker-container-after-running-it

Reply:
It seems -d and --rm can't co-exist. https://forums.docker.com/t/conflicting-options-rm-and-d/4113 I tried --rm before but it seems docker directly ignore -d flag when we added --rm.